docs/NEW_RULES.md

```diff
@@ -215,15 +219,13 @@ The `validate()` now takes an additional parameters now:
 
 Note that we don't need to define the GTFS tables as local variables and we can also omit the constructor. The notice subclass is declared the same as before.
 
-## 2. Document the new rule in [`RULES.md`](../RULES.md) and [`NOTICES.md](/docs/NOTICES.md).
+## 2. Document the new rule in [`RULES.md`](../RULES.md).
 
 Add the rule to [`RULES.md`](../RULES.md) keeping the alphabetical order of the table: 
 ```markdown
 | [NewRuleRelatedToStops](#NewRuleRelatedToStops) | new rule short description | 
 ```
-...and add a definition of that rule in the errors or warnings section (still keeping the alphabetical order).
-
-Repeat the same process in [`NOTICES.md`](/docs/NOTICES.md) to provide more details about the new notices' fields description.
+...and add a definition of that rule, the notice field descriptions, the affected filed and the reference to any additional documentation material in the errors or warnings section (still keeping the alphabetical order).
```

Comment:
Also, are we not mentioning the infos section because a contributor can't add an info notice? Just wondering.

Reply:
Good catch. The info notice is a bit unclear, we wanted to use it for metadata instead (a bit like you did with the HTML output), because we don't think the current notices are useful to people (still needs to be proven).
But in the meantime, if we want to stay consistent with RULES.md, I think we should add it. Thanks